spec/circuits/universal_batch_verifier.md

```diff
@@ -8,15 +8,15 @@ In the following, $`vk`$ will denote a Groth16 verifying key (resulting from a c
 
 The UBV circuit verifies that a set $`\{ (vk_i, \pi_i, P_i) \}_{i = 1}^{B}`$ of tuples: Groth16 verification key, Groth16 proof and public inputs, are all valid.  That is, `Groth16.Verify` $`( $vk_i, \pi_i, P_i ) ` = 1$ for each $`i = 1, \ldots, B`$.
 
-### LegoSNARK Extension
-The UBV circuit supports [LegoSNARK](https://eprint.iacr.org/2019/142)'s "commit-and-prove" extension to the original Groth16 proof system, meaning that  $`\pi`$ may contain a Pedersen commitment to certain witness values. Specifically, UBV supports up to 1 commitment to *private* witness values. Multiple Pedersen commitments and commitments to public witness values are *not* supported.
+### Extension - Commitment to witness values
+The UBV circuit supports a "commitment to witness value" extension to the original Groth16 proof system (sometimes called "commitment extension" or "Groth16 + commitment" in this document), similar but not equal to that described in [LegoSNARK](https://eprint.iacr.org/2019/142)'s.  With this extension, $`\pi`$ may contain a Pedersen commitment to certain witness values. Specifically, UBV supports up to 1 commitment to *private* witness values. Multiple Pedersen commitments and commitments to public witness values are *not* supported.
```

Comment:
We might as well point at gnark's repo, which is the ultimate source of truth when it comes to how we handle the commitment point.